scripts/tests/test_validate_docstrings.py

```diff
@@ -4,6 +4,8 @@
 import textwrap
 import pytest
 import numpy as np
+import pandas
```

Comment:
import pandas as pd

Reply:
I can change this. 

I originally wrote `import pandas` so that the names and objects on lines 1011-1014 would correspond. But better to be consistent with other usage as you suggest.